README.md

```diff
@@ -141,26 +141,30 @@ After downloading the .jar file and adding it to your CLASSPATH, add the driver
 1. Open DBeaver’s `Driver Manager` window and click `New` to add the AWS JDBC Driver.
 
 2. In `Settings`, enter the following information:
-
-```bash
-Default Port: 3306
-Driver Name: AWS JDBC Driver for MySQL
-Class Name: software.aws.rds.jdbc.mysql.Driver
-URL Template: jdbc:mysql:aws://{host}:{port}
-```
+   ```bash
```

Comment:
is this spacing conventional?

Reply:
I thought so because this is a nested code block within a list.

Reply:
Yes - i had to look at the code above that to see it